repo/packages/H/hdfs/500/resource.json

```diff
@@ -0,0 +1,59 @@
+{
+  "assets": {
+    "uris": {
+      "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz",
+      "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.12.0.tar.gz",
+      "hdfs-tar-gz": "https://downloads.mesosphere.io/hdfs/assets/hadoop-2.6.0-cdh5.16.1.tar.gz",
```

Comment:
Always specify `downloads.mesosphere.com` instead of `downloads.mesosphere.io` so that it uses our CloudFront CDN which caches the content hosted in the `downloads.mesosphere.io` S3 bucket and reduces our infrastructure (bandwidth) costs.
i.e., that URI should be:
```
"hdfs-tar-gz": "https://downloads.mesosphere.com/hdfs/assets/hadoop-2.6.0-cdh5.16.1.tar.gz",
```